handbook/security.md

```diff
@@ -255,14 +255,96 @@ We configure Chrome on company-owned devices with a basic policy.
 
 **User experience impact**
 
-* Chrome needs to be restarted within 48 hours of patches being installed. The automatic restart happens after 19:00 and before 6:00 if the computer is running, and tabs are restored (except for incognito tabs).
+* Chrome needs to be restarted within 48 hours of patch installation. The automatic restart happens after 19:00 and before 6:00 if the computer is running, and tabs are restored (except for incognito tabs).
 * Ads considered intrusive are blocked.
 * uBlock Origin is enabled by default, and is 100% configurable, improving security and performance of browsing.
 * Endpoint Verification is used to make access decisions based on the security posture of the device. For example, an outdated Mac could be prevented access to Google Drive.
 
 ### Personal mobile devices
 
-The use of personal devices is allowed for some applications, as long as the iOS or Android device is kept up to date.
+The use of personal devices is allowed for some applications, as long as the iOS or Android device
+is kept up to date.
+
+## GitHub Security
+We need to host and collaborate on code as a company making open-source software! We do this using GitHub.
+
+This section covers our GitHub configuration. Like everything we do, we aim for the right level of security and productivity.
+
+We are concerned about the integrity of the code much more than by its confidentiality, because our
+code is open-source. This is why our configuration is aimed at protecting what is in the code, but we spend no
+effort preventing "leaks", since almost everything is public anyway.
+
+### Authentication
+
+Authentication is the lynchpin of security on SaaS applications such as GitHub.
+
+GitHub authentication differs from many SaaS products in one crucial way: accounts are global. Developers can carry their accounts from company to company and use them for open source projects.
+
+Require two-factor authentication for everyone in the organization.
+We do not require SSO - as most of the software we work on is open-source and accessible to external collaborators and GitHub charges a 4x premium for it. As our security requirements increase, we will consider it, but for the moment, enforcing 2FA and code reviews is good enough.
```

Comment:
"and GitHub charges a 4x premium for it" - What is the goal with this information? Is it to promote the use of the feature because we pay for it, or do people need to be cost-conscious.

We can consider re-wording this. -"________ requires two-factor authentication for everyone in the organization. We do not require SSO - as most of the software we work on is open-source and accessible to external collaborators. GitHub charges a 4x premium for this feature."

Reply:
Thanks! Using your suggestion. The goal on the 4x is to call it out - companies that charge heavy premiums for SSSO on SaaS need to be called out, it would be illegal if it was up to me 🤣